game_patch/main/main.cpp

```diff
@@ -78,6 +78,15 @@ CodeInjection cleanup_game_hook{
     },
 };
 
+std::mt19937 rng;
```

Comment:
```suggestion
std::mt19937 g_rng;
```

Reply:
Why?

Unless there is a good reason for this, I do not plan to accept this change. It would require other changes to this PR, as well as other branches I am working on, and I'm not seeing any reason for it?

Reply:
`g_` stands for global. It is common idiom for variables having global scope. rafalh does it where `rng` is declared. Having such a short name `rng` in global scope is not good practice. It looks like a local.